features/support/route.js

```diff
@@ -171,6 +171,22 @@ module.exports = function () {
                                                               ('out' in s.intersections[0] ? s.intersections[0].bearings[s.intersections[0].out] : 0));
     };
 
+    this.lanesList = (instructions) => {
+        return this.extractInstructionList(instructions, s => {
+            return s.intersections.map( i => {
+                if(i.lanes == undefined )
```

Comment:
`if(i.lanes)` + inverting this if would be the more javascripty way (`undefined` is a `falsey` value).